pulseguardian/model/binding.py

```diff
@@ -0,0 +1,34 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+from sqlalchemy import Column, ForeignKey, Integer, String
+
+from pulseguardian.model.base import Base
+
+
+class Binding(Base):
+    __tablename__ = 'bindings'
+
+    id = Column(Integer, primary_key=True)
+    exchange = Column(String(255))
+    routing_key = Column(String(255))
+    queue_name = Column(String(255), ForeignKey('queues.name'))
```

Comment:
Looking at the db I see it is indeed storing the queue name and not the ID in the bindings table. So see two comments above. :)